### PR TITLE
C2LC-491: Updated to avoid tracking used actions separately.

### PR DESCRIPTION
**Pull request by the-t-in-rtf**
See [C2LC-491](https://issues.fluidproject.org/browse/C2LC-491) for details.

### CODE REVIEW

File: src/ProgramChangeController.js [hunk 1]
```diff
@@ -28,9 +28,9 @@ export default class ProgramChangeController {
             if (selectedAction) {
                 this.playAnnouncementForAdd(selectedAction);
                 this.doActivitiesForAdd(programBlockEditor, index);
+
```
Comment (sbates-idrc) [inline src/ProgramChangeController.js:31]: Please remove the changes to `ProgramChangeController` as they are just layout changes.

File: src/ProgramSequence.js [hunk 2]
```diff
@@ -82,4 +82,12 @@ export default class ProgramSequence {
         }
         return this.updateProgram(program);
     }
+
+    usesAction(action: CommandName): boolean {
```
Comment (sbates-idrc) [inline src/ProgramSequence.js:86]: Please add a test for this new method.

**Comment by sbates-idrc** [comment]
Please rename the PR to better reflect the change.

File: src/ProgramChangeController.js (hunk 1 above)
Comment (the-t-in-rtf) [reply src/ProgramChangeController.js]: Done.

File: src/ProgramSequence.js (hunk 2 above)
Comment (the-t-in-rtf) [reply src/ProgramSequence.js:86]: Done.

**Comment by the-t-in-rtf** [comment]
> Please rename the PR to better reflect the change.

Done.

**Pull request merged**